src/main/java/uk/gov/hmcts/reform/em/stitching/service/impl/DmStoreUploaderImpl.java

```diff
@@ -54,7 +54,7 @@ private void uploadNewDocument(File file, DocumentTask documentTask) throws Docu
                     .addFormDataPart("classification", "PUBLIC")
                     .addFormDataPart(
                         "files",
-                        documentTask.getBundle().getFileName(),
+                        documentTask.getBundle().getFileName() + ".pdf",
```

Comment:
This adds .pdf regardless of how the filename is set. Should I use 
...getFileName() + ".pdf"
or
...getFileName().concat(".pdf")
? Which one is cleaner?

Reply:
+".pdf"